cuegui/cuegui/JobMonitorTree.py

```diff
@@ -392,6 +392,10 @@ def contextMenuEvent(self, e):
         if bool(int(self.app.settings.value("AllowDeeding", 0))):
             self.__menuActions.jobs().addAction(menu, "useLocalCores")
 
+        it_view_action = self.__menuActions.jobs().addAction(menu, "viewOutput")
+        it_view_action.setDisabled(__count == 0)
+        it_view_action.setToolTip("Open ItView for the selected items")
```

Comment:
Replace "ItView" by "Viewer":

"Open Viewer for the selected items"

Reply:
```suggestion
        it_view_action.setToolTip("Open Viewer for the selected items")
```

Fixed